openproblems/tasks/label_projection/methods/mlp.py

```diff
@@ -45,7 +43,6 @@ def mlp_log_cpm(adata, test=False, max_iter=None, hidden_layer_sizes=None):
     paper_year=1990,
     code_url="https://scikit-learn.org/stable/modules/generated/"
     "sklearn.neural_network.MLPClassifier.html",
-    code_version=check_version("scikit-learn"),
```

Comment:
also here? Should the version be of the open problems function or tied to dependencies?